gulpfile.js

```diff
@@ -180,11 +181,18 @@ gulp.task('docs:mkdocs', shell.task([
   'mkdocs serve'
 ], {
   cwd: 'docs'
-}))
+}));
+
+gulp.task('lint', function () {
+  return gulp.src(['src/pathfora.js', 'gulpfile.js', 'test/pathforaSpec.js', 'docs/docs/examples/**/*.js'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
 
 gulp.task('test', ['build:styles', 'build:testjs']);
 gulp.task('build:local', ['build:styles', 'local:js']);
```

Comment:
Ok, for now I've decided to re-add it to the build task, but it doesn't run on the watcher. That way it'll run with tests. If we find a better solution later we can change it.